Cnh/add tailwind css

kmer2016 opened the pull request:
closes #50 
Each project need to import the theme.css from ui-core and tailwind-preset.js from the monorepo root.

Math-R commented on ui-core/package.json:
```diff
@@ -34,7 +34,9 @@
   "peerDependencies": {
-    "react": ">=18.0"
+    "react": ">=18.0",
+    "@osrd-project/ui-core": ">=0.0.18",
```
self dependencies ?

sim51 commented:
IMHO making a sub-project that export the tailwind config is a little overkill.
Why not just put it at the root of the project ? This file has no deps and is pretty much a simple json.

An other point, the `font.css` is not a real CSS, the `inline` function doesn't exist in CSS, it's a tailwind plugin function (from `postcss-assets`). For me it's weird to have a lib that export a CSS, which can't be used like that.

kmer2016 committed:
osrd-project: create tailwind preset at root

osrd-project: create base rollup.config generator

osrd-project: add base postcss.config
